MultiplexReservationSystem/src/main/java/p/lodz/pl/multiplexreservationsystem/model/BookedSeat.java

```diff
@@ -0,0 +1,24 @@
+package p.lodz.pl.multiplexreservationsystem.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class BookedSeat {
```

Comment:
Jaki ta klasa (encja!) ma equals? Czy to jest dobre dla JPA?

Reply:
Domyślny, tj. odziedziczony z klasy Object. Porównuje on tylko referencje obiektów, czyli sprawdza, czy dwa obiekty wskazują na to samo miejsce w pamięci. Tego oczywiście nie chcemy. W tym przypadku mogłoby dojść do sytuacji, w której zarezerwowane miejsce zostałyby nadpisane w bazie danych, ponieważ my porównujemy po referencji (equals() zwróci false), a JPA w bazie danych bo kluczu głównym.